packages/babel-core/src/transformation/file/index.js

```diff
@@ -427,6 +427,8 @@ export default class File extends Store {
       );
       traverse(this.ast, visitor, this.scope);
 
+      t.validateDeep(this.ast);
```

Comment:
curious how much slower it gets when doing this (I guess it's 1 one time check)?